practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/configuration/PractitionerConfigurationProvider.java

```diff
@@ -1,38 +1,58 @@
 package uk.ac.ox.ndph.mts.practitioner_service.configuration;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
-
 import org.springframework.beans.factory.annotation.Value;
-import org.springframework.core.io.Resource;
+import org.springframework.http.MediaType;
 import org.springframework.stereotype.Component;
-
+import org.springframework.web.reactive.function.client.WebClient;
+import org.springframework.web.reactive.function.client.WebClientException;
+import uk.ac.ox.ndph.mts.practitioner_service.exception.RestException;
 import uk.ac.ox.ndph.mts.practitioner_service.model.PractitionerConfiguration;
-import uk.ac.ox.ndph.mts.practitioner_service.exception.InitialisationError;
+import java.io.File;
```

Comment:
is this needed just for the File.separator?

Reply:
yes, Its needed for the separator